src/binding.cc

```diff
@@ -1,53 +1,17 @@
 #include <node.h>
 #include <node_buffer.h>
+#include <node_object_wrap.h>
 #include <stdlib.h>
 #include <stdio.h>
 #include <string.h>
 #include <string>
 #include <mysql.h>
+#include <nan.h>
 
 using namespace node;
 using namespace v8;
 
 static Persistent<FunctionTemplate> Client_constructor;
-static Persistent<String> emit_symbol;
-static Persistent<String> connect_symbol;
-static Persistent<String> resquery_symbol;
-static Persistent<String> resabort_symbol;
-static Persistent<String> resdone_symbol;
-static Persistent<String> qrow_symbol;
-static Persistent<String> qrowerr_symbol;
-static Persistent<String> qerr_symbol;
-static Persistent<String> qabort_symbol;
-static Persistent<String> qdone_symbol;
-static Persistent<String> code_symbol;
-static Persistent<String> err_symbol;
-static Persistent<String> close_symbol;
-static Persistent<String> insert_id_symbol;
-static Persistent<String> affected_rows_symbol;
-static Persistent<String> num_rows_symbol;
-static Persistent<String> cfg_user_symbol;
-static Persistent<String> cfg_pwd_symbol;
-static Persistent<String> cfg_host_symbol;
-static Persistent<String> cfg_port_symbol;
-static Persistent<String> cfg_socket_symbol;
-static Persistent<String> cfg_db_symbol;
-static Persistent<String> cfg_timeout_symbol;
-static Persistent<String> cfg_secauth_symbol;
-static Persistent<String> cfg_multi_symbol;
-static Persistent<String> cfg_compress_symbol;
-static Persistent<String> cfg_metadata_symbol;
-static Persistent<String> cfg_ssl_symbol;
-static Persistent<String> cfg_ssl_key_symbol;
-static Persistent<String> cfg_ssl_cert_symbol;
-static Persistent<String> cfg_ssl_ca_symbol;
-static Persistent<String> cfg_ssl_capath_symbol;
-static Persistent<String> cfg_ssl_cipher_symbol;
-static Persistent<String> cfg_ssl_reject_symbol;
-static Persistent<String> cfg_local_infile_symbol;
-static Persistent<String> cfg_read_default_file;
-static Persistent<String> cfg_read_default_group;
-static Persistent<String> cfg_charset_symbol;
```

Comment:
Creating a new string when emitting events can be a huge performance hit (for example: when you have a *lot* of rows). This is why I was using Persistent strings for event names and other frequently used strings.

The need for config-related Persistent strings might be debatable though, I haven't benchmarked with and without.